config/schedule.rb

```diff
@@ -9,30 +13,36 @@ def stagger(hour)
 
 # bi-weekly harvest at 5pm in qa, on the 8th and 23rd of the month
 every "0 17 7,23 * *", roles: [:harvester_qa] do
-  rake 'harvest:all_authors_update'
+  set :check_in, Settings.honeybadger_checkins.harvest_all_authors
+  rake_hb 'harvest:all_authors_update'
 end
```

Comment:
This task looks like it only runs on QA. If so, do we want it to check in?

Reply:
Yes, I think it makes sense to have the check-ins for QA as well since the Profiles team can use that environment for integration testing (and this helps keep the data up to date).  

Note that I created a check in key for all QA environments and even though there is a separate line in the `schedule.rb` file, it is actually calling the exact same rake task (just on a different frequency), so I think this is ok (and ditto for the other rake tasks on QA).

dev is being skipped on purpose (no checkin neeed, it has no hb key)

Reply:
okie dokie, thanks